frontend-react/src/components/CreateEventForm/AddEmail.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {useEffect, useState} from "react";
+
+
+type AddEmailProps = {
+    emails: string[];
+    addEmail: (email: string) => void;
+}
+
+export const AddEmail = ({emails, addEmail}: AddEmailProps) => {
+
+    const [email, setEmail] = useState('');
+
+    return (
+        <form>
+            <div>{emails.map((e, i) => <div key={i}>{e}</div>)}</div>
+            <input type='text' placeholder='email' value={email} onChange={(e) => setEmail(e.target.value)}/>
```

Comment:
Add `TODO` here. In the future we will need to validate if the text is an email

Reply:
added